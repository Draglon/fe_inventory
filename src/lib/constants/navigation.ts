import {
  ordersRoute,
  groupsRoute,
  productsRoute,
  usersRoute,
  settingsRoute,
} from "@/lib/routes";

export const NAVIGATION_MENU = [
  {
    href: ordersRoute,
    text: "Navigation.orders",
  },
  {
    href: groupsRoute,
    text: "Navigation.groups",
  },
  {
    href: productsRoute,
    text: "Navigation.products",
  },
  {
    href: usersRoute,
    text: "Navigation.users",
  },
  {
    href: settingsRoute,
    text: "Navigation.settings",
  },
];
