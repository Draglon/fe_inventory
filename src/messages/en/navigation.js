const navigation = {
  "Navigation": {
    "parishes": "Parishes",
    "groups": "Groups",
    "products": "Products",
    "users": "Users",
    "settings": "Settings"
  },
};

export default navigation;