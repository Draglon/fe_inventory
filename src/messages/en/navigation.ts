const navigation = {
  "Navigation": {
    "orders": "Orders",
    "groups": "Groups",
    "products": "Products",
    "users": "Users",
    "settings": "Settings"
  },
};

export default navigation;