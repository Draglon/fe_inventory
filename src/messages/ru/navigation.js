const navigation = {
  "Navigation": {
    "orders": "Приходы",
    "groups": "Группы",
    "products": "Продукты",
    "users": "Пользователи",
    "settings": "Настройки"
  },
};

export default navigation;