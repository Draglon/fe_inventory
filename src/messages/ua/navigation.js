const navigation = {
  "Navigation": {
    "orders": "Замовлення",
    "groups": "Групи",
    "products": "Продукти",
    "users": "Користувачі",
    "settings": "Налаштування"
  },
};

export default navigation;