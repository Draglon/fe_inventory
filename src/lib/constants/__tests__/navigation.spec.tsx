import * as navigation from "../navigation";

it("navigation constants match snapshot", () => {
  expect(navigation).toMatchSnapshot();
});
