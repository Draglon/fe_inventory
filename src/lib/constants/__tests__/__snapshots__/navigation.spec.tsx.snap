// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`navigation constants match snapshot 1`] = `
{
  "NAVIGATION_MENU": [
    {
      "href": "/orders",
      "text": "Navigation.orders",
    },
    {
      "href": "/groups",
      "text": "Navigation.groups",
    },
    {
      "href": "/products",
      "text": "Navigation.products",
    },
    {
      "href": "/users",
      "text": "Navigation.users",
    },
    {
      "href": "/settings",
      "text": "Navigation.settings",
    },
  ],
}
`;
